build_projects/Microsoft.DotNet.Cli.Build.Framework/Enumerations/BuildArchitecture.cs

```diff
@@ -3,6 +3,7 @@ namespace Microsoft.DotNet.Cli.Build.Framework
     public enum BuildArchitecture
     {
         x86 = 1,
-        x64 = 2
+        x64 = 2,
+        arm = 3
```

Comment:
Sure, I added arm64. But I do not add `Isarm64()` because I don't know what value you use when you build arm64 modules.